packages/lb-annotation/src/core/pointCloud/matrix.ts

```diff
@@ -64,6 +65,67 @@ export function rotatePoint(
   return pointVector.clone().applyMatrix4(TBack).applyMatrix4(Rz).applyMatrix4(TFrom);
 }
 
+/**
+ * Check if the matrix is valid
+ * @param matrix
+ * @param numRows
+ * @param numColumns
+ * @returns
+ */
+export function isMatrixValid(matrix: number[][], numRows: number, numColumns: number): boolean {
+  // Check if the number of rows and columns match the expected values
+  if (matrix.length !== numRows || matrix.some((row) => row.length !== numColumns)) {
+    return false;
+  }
+
+  return true; // Compliant with the rule
+}
+
+/**
+ * Check if the calib is in fisheyeCalib.
+ * @param calib
+ * @returns
+ */
+export const isFisheyeCalibValid = (calib: ICalib) => {
+  if (calib.fisheyeDistortion?.length > 0 && isMatrixValid(calib.P, 3, 4) && isMatrixValid(calib.T, 3, 4)) {
+    return true;
+  }
+
+  return false;
+};
+
+export const lidar2ImageByFisheye = (point: I3DSpaceCoord, calib: ICalib) => {
```

Comment:
How aboout renaming as lidarToFishEyeCood or lidar2FishEye2D? 
Add comment and returns.

Reply:
You are right. I think `lidar2FisheyeImage` is better.